apps/payments/next/next.config.js

```diff
@@ -12,6 +12,9 @@ const nextConfig = {
     // See: https://github.com/gregberge/svgr
     svgr: false,
   },
+  experimental: {
+    serverComponentsExternalPackages: ['knex'],
```

Comment:
The payments-next build started failing after including @fxa/shared/db/mysql/account in the import stack. A quick google suggested this as a fix.

Reply:
Would you happen to have the link on hand? Would be really curious to read it. 😄

Reply:
I believe this is where I eventually found the answer.

https://github.com/vercel/next.js/issues/52091

Some more info can also be found here in the Next.js docs

https://nextjs.org/docs/app/api-reference/next-config-js/serverComponentsExternalPackages